src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/MarkupCompiler/MarkupCompiler.cs

```diff
@@ -483,7 +483,7 @@ private void Initialize(FileUnit sourceFile)
                 }
 
                 int pathEndIndex = SourceFileInfo.RelativeSourceFilePath.LastIndexOf(Path.DirectorySeparatorChar);
-                string targetPath = string.Concat(TargetPath, SourceFileInfo.RelativeSourceFilePath.AsSpan(0, pathEndIndex + 1));
+                string targetPath = string.Concat(TargetPath, SourceFileInfo.RelativeSourceFilePath.Substring(0, pathEndIndex + 1));
```

Comment:
Why we back to `Substring` here? Will it cause the warning?

Reply:
Actually, it wasn't raising a warning, so the change was unnecessary.